src/CMakeLists.txt

```diff
@@ -124,3 +124,12 @@ if(KOMPUTE_OPT_INSTALL)
         DESTINATION lib/cmake/kompute)
 endif()
 
+if(KOMPUTE_OPT_BUILD_PYTHON)
+    include_directories(${PROJECT_SOURCE_DIR}/python/pybind11/include)
+    if (WIN32)
+        execute_process(COMMAND CMD /c python -c "from distutils.sysconfig import get_python_inc; print(get_python_inc())" OUTPUT_VARIABLE PYTHON_INCLUDE_DIR)
+    else()
+        execute_process(COMMAND python -c "from distutils.sysconfig import get_python_inc; print(get_python_inc())" OUTPUT_VARIABLE PYTHON_INCLUDE_DIR)
+    endif()
+    include_directories(${PYTHON_INCLUDE_DIR})
```

Comment:
It seems the Python build is failing (fantastic to see the CI jobs making the workflows much simpler) - it seems like the Python headers are not in place.

Is there a reason why you are running this like this with `execute_process`? It may be possible to just do a `find_package` for the Python headers instead.

Reply:
`find_package` did not work on my machine for some reason. Could you check if `python-dev` is installed in the Docker image or install it during the build?

Reply:
Good point! Let me have a look - I also had a similar experience importing the python headers, let me rerun with python-dev and also let me see if I can find the snippet with  `find_package`.

Reply:
@alexander-g Python3-dev is already installed, would this require python-dev instead?

Reply:
Ok looking at the code above you need to run `python3` instead of `python`

Having said that it would probably best to approach it with the find_package, I have found an option below - does that work for you?
```
find_package(PythonLibs REQUIRED)
include_directories(${PYTHON_INCLUDE_DIRS})
```

Reply:
Ok I just tested with the snippet above (ie `find_package(PythonLibs REQUIRED)`) and all tests seem to pass, so that should do

Reply:
Awesome - all tests pass 👍

Reply:
Yes, it works on my device too now. Thanks for fixing it. I think I only tried `find_package(Python)` without thinking too much and that did not work

Reply:
Great - no worries, I first tried `find_package(Python)` as well and it didnt' work, but found the answer above in a stackoverflow page that covered it in more detail. I also tried your code with `python3` instead of `pyhton` and that fixed it as well